schema/rusk.proto

```diff
@@ -23,16 +20,14 @@ message ExecuteStateTransitionRequest {
     repeated Transaction txs = 1;
     fixed64 height = 2;
     bytes seed = 3;
+    bool finalized = 4;
+    bytes base_state_root = 5;
```

Comment:
I would call it `state_root` as for the other, since it's the same field (it's the `state_root` at the time of the transition execution, `base_state_root` might be misleading, since it might lead to think is a different thing and /or used only for (non) finalized transitions.

Also, we should have only `StateTransitionRequest` since the field should be the same for both `ExecuteStateTransition` and `VerifyStateTransition`.

They might have different return values, but I would double check with @toghrulmaharramov (e.g. I've no idea honestly what `failed_calls` should be here. Ideally both `Execute` and `Verify` does the same thing, with the only difference that the first actually change the state, the other one discard it after).

Reply:
Good catch, updated

Reply:
I think in the past the `VerifyStateTransition` was created to split the verification that could be performed faster by only verifying the proofs - but that is likely not applicable here